packages/components/src/template-registry.ts

```diff
@@ -3,6 +3,9 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+import type HdsAccordionIndexComponent from './components/hds/accordion';
```

Comment:
See comment here https://github.com/hashicorp/design-system/pull/2087/files#r1602930931

This tells me is better to decide sooner than later ;)

Reply:
Indeed – let's align on this. As I mentioned on the other thread, we may need to align on class names as well

Reply:
Addressed in [3775a5a](https://github.com/hashicorp/design-system/pull/2101/commits/3775a5adc609b74d069e056da20ede9f42fafbe8) as discussed (removed `Index`)